packages/fastify/src/types.ts

```diff
@@ -0,0 +1,3 @@
+import type { ClerkOptions } from '@clerk/backend';
+
+export type ClerkFastifyOptions = Omit<ClerkOptions, 'apiKey'>;
```

Comment:
🙃 to avoid maintenance of this type on changes i would suggest 1 of the following: 
```
//1.
export type { ClerkOptions as ClerkFastifyOptions} from '@clerk/backend';
// 2. or don't even define a separate ClerkFastifyOptions and use ClerkOptions 
```

Reply:
Valid point Dimitri; we cannot remove `apiKey` from the common backend type right now - otherwise, we will break backward compatibility.

In general, though, I like separate high-level types on a per-framework basis because it's easy to make fine-grained adjustments later without introducing more types.